coordinator/tests/services/translation/response/release/release-response.spec.ts

```diff
@@ -29,7 +29,17 @@ import {Organization as IOrgansation} from "../../../../../../models/fhir/practi
 
 describe("outer bundle", () => {
   describe("passed prescriptions", () => {
-    const result = translateReleaseResponse(getExamplePrescriptionReleaseResponse("release_success.xml"))
+    const logger = {
```

Comment:
Hi @MatthewPopat-NHS, to reduce repeating ourselves we could pull the mock logger out of the each describe block. 

We could have a file responsible for mock loggers which can then be used by more tests in the future

const mockErrorLogger =  {
      level: "error",
      info: jest.fn(),
      error: jest.fn(),
      warn: jest.fn(),
      debug: jest.fn(),
      fatal: jest.fn(),
      trace: jest.fn(),
      silent: jest.fn()
    }

If someone needs an mockInfoLogger for a test in future they can added 


const _mockinfoLogger_ =  {
      level: "info",
      info: jest.fn(),
      error: jest.fn(),
      warn: jest.fn(),
      debug: jest.fn(),
      fatal: jest.fn(),
      trace: jest.fn(),
      silent: jest.fn()
    }

Or even a mockLogerFactory method that creates the mock logger with which ever level we need